pkgs/development/python-modules/tinyobjloader-py/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, pybind11, tinyobjloader }:
+
+buildPythonPackage rec {
+  pname = "tinyobjloader-py";
+  inherit (tinyobjloader) version src;
+
+  # Build needs headers from ${src}, setting sourceRoot or fetching from pypi won't work.
+  preConfigure = ''
+    cd python
+  '';
+
+  buildInputs = [ pybind11 ];
+
+  # No tests included upstream
+  doCheck = true;
+  pythonImportsCheck = [ "tinyobjloader" ];
+
+  meta = with lib; {
+    inherit (tinyobjloader) meta;
```

Comment:
This causes this package to fail to evaluate correctly:

```
	 - key 'meta' is unrecognized; expected one of: 
	      ['available', 'badPlatforms', 'branch', 'broken', 'changelog', 'description', 'downloadPage', 'downloadURLRegexp', 'executables', 'homepage', 'hydraPlatforms', 'insecure', 'isBuildPythonPackage', 'isFcitxEngine', 'isGutenprint', 'isIbusEngine', 'knownVulnerabilities', 'license', 'longDescription', 'maintainers', 'name', 'outputsToInstall', 'platforms', 'position', 'priority', 'repositories', 'schedulingPriority', 'tag', 'tests', 'timeout', 'unfree', 'unsupported', 'updateWalker', 'version'], refusing to evaluate.
```

I think you probably meant something like

```
meta = with lib; tinyobjloader.meta // {
  ...
};
```

instead?

Reply:
That works, thank you!